lib/run.js

```diff
@@ -155,9 +160,10 @@ function run (opts, cb) {
       let result = {
         title: opts.title,
         url: opts.url,
-        requests: histAsObj(requests, totalRequests),
+        requests: histAsObj(requests, totalCompletedRequests),
         latency: addPercentiles(latencies, histAsObj(latencies)),
         throughput: histAsObj(throughput, totalBytes),
+        totalRequests: totalRequests,
```

Comment:
I would put it inside the requests object, as `sent`. Does it make sense?

@davidmarkclements ?